flex/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN cd /root && \
 
 RUN cp /usr/local/lib/libgrape-lite.so /usr/lib/libgrape-lite.so
 
-RUN git clone https://github.com/alibaba/hiactor.git && cd hiactor && git checkout e16949ca53 && \
+RUN git clone https://github.com/alibaba/hiactor.git && cd hiactor && git checkout 8add4b330c31f8a47d83c5804072a8d42d10d32d && \
```

Comment:
You say "update libgrape-lite" in title but actually you are bumping up the hiactor revision?

Reply:
Oops, my bad. :)